### PR TITLE
Add - remove task

### PR DESCRIPTION
**Pull request by Obote**
### **What does this PR do?**

- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @Obote, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not a summary describing in more detail the changes introduced in the PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.html [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
+            <div class="taskDetails">
+              <input type="checkbox" class="task-check" />
+              <label class="taskTitle">Hello world</label>
+            </div>
+
+            <div class="taskIcon">
+              <ion-icon class="taskIconEdit" name="create-outline"></ion-icon>
+              <ion-icon class="taskIconDelete" name="trash-outline"></ion-icon>
+            </div>
+          </div> -->
```
Comment (joskalenda) [inline src/index.html]: - [ ] If you think this piece of code is not relevant to the milestone, please make sure you remove it or uncomment it as it makes your code dirty and unprofessional, @Obote

File: src/localstorage.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+function LS() {}
+LS.prototype.fetchItem = function () {
+  let items = localStorage.getItem('items');
+  if (items) {
+    items = JSON.parse(items);
+  } else {
+    items = [];
+  }
+  return items;
```
Comment (joskalenda) [inline src/localstorage.js]: - [ ] I think you need to take a close look at this record, the first issue is, the user is able to insert empty todo and the second one is, that anytime the user refreshes the app, all the data is lost. please work on fixing it

File: README.md
Comment (joskalenda) [inline README.md:1]: - [ ] Please make sure you update your readme with the correct MIT license as for now, the [link](https://github.com/Obote/todo-list/blob/add-remove-task/LICENSE) to it is broken

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Obote , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Obote , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**